src/pages/registration/resource_type_tab/components/SearchContainerField.tsx

```diff
@@ -50,7 +51,7 @@ const SearchContainerField = (props: SearchContainerFieldProps) => {
       : searchContainerOptions?.hits ?? [];
 
   return (
-    <>
+    <div data-testid={`${props.dataTestId}`}>
```

Comment:
```suggestion
    <div data-testid={props.dataTestId}>
```
mulig denne kunne vært lagt rett på `Autocomplete` også, så slipper vi å nøste inn en ekstra div

Reply:
hvis den ligger i Autocomplete mister vi summary da, så det er kanskje ikke bedre likevel